jobs/integr8ly/poc-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,58 @@
+---
+
+- job:
+    name: poc-testing-nightly-trigger
+    project-type: pipeline
+    description: "Nightly pipeline trigger for QE POC cluster testing."
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H H(0-3) * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            GH_CLIENT_ID=yourclientid
+            GH_CLIENT_SECRET=yourclientsecret
+            SELF_SIGNED_CERTS=false
+            RECIPIENTS=integreatly-qe@redhat.com
+            INSTALLATION_REPOSITORY=https://github.com/integr8ly/installation.git
+            INSTALLATION_BRANCH=master
+            TEST_SUITES_REPOSITORY=https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            TEST_SUITES_BRANCH=master
+            CLUSTER_URL=https://integreatly-qe.opentry.me
+            MASTER_URLS=ec2-54-154-4-117.eu-west-1.compute.amazonaws.com,ec2-34-244-30-214.eu-west-1.compute.amazonaws.com,ec2-34-241-180-41.eu-west-1.compute.amazonaws.com
```

Comment:
I think master URLs might be outdated after recent changes. Also (if my PR is merged) there will be one additional parameter required: `TO_DO=full`.

Reply:
yeah sure....

Reply:
Updated to the new cluster